docs/application/native/guides/text-input/autofill-service.md

```diff
@@ -0,0 +1,335 @@
+# Autofill Service
+
+Autofill Service application is a service application to store the user input and provides the data previously entered by the user. Developers can implement the Autofill service in their own application to serve the Autofill information.
+
+The main features of the Autofill service API include:
+
+- Authentication Information
+
+  You can [receive the request of authentication, and send the authentication information](#authentication-information).
+
+- Autofill fill request and autofill fill response
```

Comment:
Receiving and sending autofill request and response

Reply:
According to "no gerund" guide, I use "Receive autofill fill request and send autofill fill response".

Reply:
@imfine98, sure I agree with your opinion.